core/src/com/po/fuck/model/Player.java

```diff
@@ -32,8 +34,10 @@ public class Player extends Entity {
     private final Managed<KeyboardController> keyboardController = Manager.create(new KeyboardController(this));
     private final Managed<MouseController> mouseController = Manager.create(new MouseController(this));
 
-    Player(GeometryData geometryData) {
-        super(geometryData, PLAYERS_HEALTH);
+    Player(PositionData positionData) {
+        super(new GeometryData(positionData,
+                BasicSpriteManager.getBasicSpriteInfo(Player.class).getSize()),
```

Comment:
Maybe move `BasicSpriteManager.getBasicSpriteInfo(Player.class)` into a variable, not critical though.

Reply:
It's inside ```super(...)```, so there shouldn't be anything above it.

Reply:
You can make constructor private, and add `static Player create(...) { var info = BasicSpriteManager.getBasicSpriteInfo(Player.class); return Player(...); }`
Not critical though